superset-frontend/src/SqlLab/main.less

```diff
@@ -259,6 +259,22 @@ div.Workspace {
     &:active {
       background: none;
     }
+
+    .fa.fa-close {
```

Comment:
I don't see the font awesome class applied anywhere, does this actually do something?

If you need to add styling to the Icon, I'd recommend making a oneoff element in the `TabbedSqlEditors` file like so:
```typescript
const CloseIcon = styled(Icon)`
  // add css styles here

  .or-add-a-class {
    // more styles
  }
`;
```

However, it looks like this styling is specifically to make the Icon interactable. In that case, to ensure a11y, you should wrap it in an `a` tag, a `button` tag, or some other element with the appropriate aria role applied, similar to here: https://github.com/apache/incubator-superset/commit/80b06f682727338fc6ce7e804466620cffbcc13c#diff-ce4d2524d6740fe3a5ac128b5e33baa4R382

On a side note, we should create an `IconButton` component that abstracts all this a11y stuff away and replaces the current uses of bare `Icon`s that are clickable. @nytai @lilykuang or @rusackas, would any of you be interested in this? Or should I take it on?

Reply:
since when a11y is a requirement?

Reply:
not a requirement, but certainly something we should consider i think! I can't even imagine how bad it is to use Superset with a screenreader or keyboard controls....

Reply:
correct. this PR can't handle big issue like that. no one will care there is an icon in whole Superset is a11y. yay!!

Reply:
this was introduced by last try (to use font awsome icon). fixed

Reply:
could you at least add `role="button" tabIndex={0}` to the `Icon` component? That should be sufficient here

Reply:
I can do change for this place. but you probably have to fix all other places...

Reply:
yup, i'll fix other stuff in the future, likely by adding a lint rule